 \documentclass[25pt, a0paper, portrait, margin=0mm, innermargin=15mm,
     blockverticalspace=15mm, colspace=15mm, subcolspace=8mm]{tikzposter} %Default values for poster format options.
     
\usepackage{tabularx,booktabs,adjustbox} % For tables
\usepackage{wrapfig,anyfontsize,lipsum}
\linespread{1.2}
\usetikzlibrary{calc,fit,arrows,decorations.pathmorphing,backgrounds,fit,positioning,patterns}
\usetikzlibrary{shapes.symbols}

\usepackage{color,colortbl}
\definecolor{Cblack}{rgb}{0,0,0}
\definecolor{Corange}{rgb}{0.9,0.6,0}
\definecolor{Cskyblue}{rgb}{0.35,0.7,0.9}
\definecolor{Cbluegreen}{rgb}{0,0.6,0.5}
\definecolor{Cyellow}{rgb}{0.95,0.9,0.25}
\definecolor{Cblue}{rgb}{0,0.45,0.7}
\definecolor{Cvermillion}{rgb}{0.8,0.4,0}
\definecolor{Cpurple}{rgb}{0.8,0.6,0.7}

\newcommand{\Cblack}[1]{\textcolor{Cblack}{#1}}
\newcommand{\Corange}[1]{\textcolor{Corange}{#1}}
\newcommand{\Cskyblue}[1]{\textcolor{Cskyblue}{#1}}
\newcommand{\Cbluegreen}[1]{\textcolor{Cbluegreen}{#1}}
\newcommand{\Cyellow}[1]{\textcolor{Cyellow}{#1}}
\newcommand{\Cblue}[1]{\textcolor{Cblue}{#1}}
\newcommand{\Cvermillion}[1]{\textcolor{Cvermillion}{#1}}
\newcommand{\Cpurple}[1]{\textcolor{Cpurple}{#1}}

 \newcommand{\magenta}[1]{\textcolor{magenta}{#1}}
 \newcommand{\teal}[1]{\textcolor{teal}{#1}}
  \newcommand{\gray}[1]{\textcolor{gray}{#1}}

% tikz colour settings
\tikzset{pop0/.style={red!50!yellow},pop1/.style={violet!80},pop2/.style={olive!70!green}}

\definecolorpalette{myColorPalette} {
\definecolor{colorOne}{named}{teal}
\definecolor{colorTwo}{named}{white}
\definecolor{colorThree}{named}{cyan}
}

 \tikzposterlatexaffectionproofoff %shows small comment on how the poster was made at bottom of poster

 % Commands
 \newcommand{\bs}{\textbackslash}   % backslash
 \newcommand{\cmd}[1]{{\bf \color{red}#1}}   % highlights command
% \newcommand{\crossmark}{\ding{55}}

 % Title, Author, Institute
 \title{{\bf Analysis, inference and simulation of identity-by-descent in large datasets}}
 \author{{\bf Georgia Tsambos}}
 \institute{{\bf University of Melbourne, Australia}}

 % -- PREDEFINED THEMES ---------------------- %
 % Choose LAYOUT:  Default, Basic, Rays, Simple, Envelope, Wave, Board, Autumn, Desert,
 \usetheme{Autumn}
\usecolorstyle[colorPalette=myColorPalette]{Denmark}


 \begin{document}

     \maketitle
\block[bodyverticalshift=-2cm]{}{
   {\large\teal{
 Georgia Tsambos (1, 2), Peter Ralph (3), Jerome Kelleher (4), Stephen Leslie (1, 2, 5), Damjan Vukcevic (1, 2).
(1) School of Mathematics and Statistics, University of Melbourne, Australia (2) Melbourne Integrative Genomics, University of Melbourne, Australia, (3) Department of Mathematics, University of Oregon, United States, (4) Big Data Institute, University of Oxford, United Kingdom, (5) School of Biosciences, University of Melbourne, Australia. Presenting author: gtsambos (at) student.unimelb.edu.au}
}
}
   
   %%% BLOCK 0  
 \block[titleoffsety=1cm,bodyverticalshift=-20cm]{0. Introduction}{
 }
 \begin{columns}
 \begin{subcolumns}
   \subcolumn{31} \block[bodyoffsetx=1.5cm,bodyverticalshift=-5cm]{}{
{\fontsize{34}{35}\selectfont Identity-by-descent (IBD) analyses make it possible to study the relatedness of individuals on a detailed scale without invoking murky concepts of population labels. 
This poster showcases a recent advance in the tskit [1] software, the \texttt{find-ibd} method, which allows complete IBD information to be calculated efficiently on large samples at all time scales. 
The utility of this tool will be demonstrated with simulations, as well as with data from the Human Genome Diversity Project.}\\[3mm] 
    }
 \end{subcolumns}
 \end{columns}

%%% BLOCK 1
     \begin{columns}%blocks will be placed into columns
         \column{.5}
         \block[roundedcorners=40,titleoffsety=-3cm,bodyoffsety=-3cm,bodyverticalshift=-1cm]{1. Data structures and definitions}{

{\fontsize{34}{35}\selectfont The {\bf\magenta{tree sequence}} data structure [1,2] encodes a complete genealogy for a sample of chromosomes in a succinct set of tables. Compared with traditional sequence-based formats, tree sequences are \emph{compact}, \emph{fast} to process and \emph{informative} of the history of the sample.

\vspace{10mm}

\begin{center}
\input{pics/ibd-defn.tex}
\end{center}

\vspace{-80mm}

In diploid species, each position of an organism's genome is inherited from one of the two genomes held by each of the parental individuals.
Two segments of DNA in different chromosomes that have been inherited whole (ie. unbroken by recombination) from the same ancestral chromosome are said to be {\bf \magenta{ identical by descent}}, or {\bf \magenta{IBD}}.
An illustration of this process is shown above.
}
     }
     
     %%%% BLOCK 3
     \block[titleoffsety=1cm,bodyoffsety=5cm]{3. Example: total IBD sharing over time}{
     } 
\centering
  \begin{subcolumns}

  \subcolumn{.25}
   \block{}{
%   \centering
   \includegraphics[scale=.63]{pics/kinships-10000.png}
   \includegraphics[scale=.63]{pics/kinships-8000.png}
    \includegraphics[scale=.63]{pics/kinships-6000.png}
    \includegraphics[scale=.63]{pics/kinships-4000.png}
\includegraphics[scale=.63]{pics/kinships-2000.png}
  }
   \subcolumn{.6}
  \block{}{
%  \centering
  \input{pics/demographic-example.tex}
  }
    \colorlet{notebgcolor}{teal!30}
   \note[targetoffsetx=0cm, targetoffsety=+11cm,width=.2\textwidth,innersep=.4cm]{
Using \texttt{msprime} [2], I simulated a three-population demographic scenario, sampling twelve 100Mb chromosomes from each of the three populations. I assumed a recombination rate of $1\times 10^{-8}$ base/generation.
The five heatplots on the left show the total proportion of IBD sharing between each pair of simulated samples with respect to ancestors from five different time points. 
The simulation took $\approx 6$ sec to run, and IBD information was extracted in a further $\approx 2$ sec.
   }
  \end{subcolumns}



     \column{.5}
     %%% BLOCK 2
         \block[titleoffsety=-3cm,bodyoffsety=1.5cm]{2. The find-ibd algorithm}{}  
%          \begin{subcolumns}       
%         \subcolumn{.45}
          \block{}{
The \texttt{find-ibd} algorithm is implemented in C and Python in \texttt{tskit}. 
For each pair of sample haplotypes $(n_1, n_2)$ in the tree sequence, it outputs IBD records $(l, r; a)$ iff both $n_1$ and $n_2$ have inherited the interval $[l, r)$ from ancestor $a$ along the same genealogical path.
Consider the following tree sequence:
\vspace{-10mm}
\begin{center}
\input{pics/tree-sequence.tex}
\end{center}
The algorithm returns the following IBD records for sample pair $(0, 5)$:
\[
\left( 89, 100; 8\right),
\left( 54, 89; 12\right),
\left( 0, 54; 12 \right)  
\]
You can make \texttt{find-ibd} more efficient by specifying a maximum age or minimum length for the IBD segments you're interested in. If \texttt{min-length=20}, the records are
\[
\left( 54, 89; 12\right),
\left( 0, 54; 12 \right)  
\]
If \texttt{max-time=50}, the only record is
\[
\left( 89, 100; 8\right)
\]
Since the IBD records contain information about the common ancestor, they can be dated to a particular time.
\texttt{find-ibd} requires a single pass over the edge table of the tree sequence, and each edge must only be processed once, regardless of how many gene trees it appears in.
This makes it extremely efficient.
          }           
       
\vspace{20cm}

             \block[titleoffsety=0cm,bodyoffsety=0cm]{4. Application: demography inference}{

\begin{adjustbox}{valign=b}
\begin{minipage}[t]{0.5\linewidth}
{\small
\begin{tabularx}{.8\textwidth}{p{9cm}X}
\toprule
\multicolumn{2}{c}{\bf Model inference}\\
\midrule
Posterior probability for correct~model &  $96.33\% \pm 2.68\%$ \\
Votes for correct demography &$93.36\% \pm 4.69\%$\\
Prior error rate & $0.0993 \pm 0.0008$ \\
\bottomrule
\end{tabularx}
}
\end{minipage}\end{adjustbox}\begin{adjustbox}{valign=c}
\begin{minipage}[t]{0.35\linewidth}
\includegraphics[scale=.2]{pics/abc-ibd-4.png}
\end{minipage}
\end{adjustbox}

{\fontsize{34}{35}\selectfont To explore the power of our method, we attempted to recreate some of the important findings from a recent study of Ashkenazi Jewish (AJ) demographic history [3]. This work provided evidence for a recent divergence event between Eastern and Western communities of AJ people. It was estimated that this happened about 15 generations ago.\\[2mm]
Closely following [3], we performed 100 000 simulations of various demographic scenarios, and 20 simulations under the parameter values inferred by [3], which we treated as proxies for the truth. For each simulation, we calculated moments of IBD segment lengths at multiple time points. We used the \texttt{abcrf} package [4] to infer the most plausible demographic scenario, and the \texttt{abc} package [5] with neural-net regression to estimate parameter values.\\
All simulations and analyses ran on a standard desktop in $<12$ hours.
}
}
     \end{columns}

%%% BLOCK 5
 \block[titleoffsety=1.2cm,bodyoffsety=1.2cm]{5. Acknowledgements, references and further information}{
 }
  \begin{columns}
 \begin{subcolumns}
 \renewcommand\baselinestretch{1}\selectfont
    \subcolumn{22} \block[bodyoffsetx=1.5cm,bodyverticalshift=-5cm]{}{\footnotesize
     GT is funded by the Helen Freeman scholarship, the Maurice Belz Fund and the Australian Government's Research Training Scheme.\\[1mm]
     [1] \texttt{https://tskit.readthedocs.io/en/latest/}\\[1mm]
     [2] Kelleher, J., et al. (2016). Efficient Coalescent Simulation and Genealogical Analysis for Large Sample Sizes. PLOS Computational Biology, 12(5).\\[1mm]
     [3] Gladstein, A and Hammer, M. (2019). Substructured Population Growth in the Ashkenazi Jews Inferred with Approximate Bayesian Computation. Molecular Biology and Evolution, 36(6), 1162-1171.\\[1mm] 
    [4] Raynal, L., et al. (2019). ABC random forests for Bayesian parameter inference. Bioinformatics, 35(10), 1720 - 1728.\\[1mm]
	[5] Csillery, K., et al. (2012). Abc: An R package for approximate Bayesian computation (ABC). Methods in Ecology and Evolution, 3(3), 475 - 479.
     }

       \subcolumn{3} \block[bodyoffsetx=-3cm,bodyverticalshift=-5.7cm]{}{
       \begin{center}
       {\small Come say hi!}\\
       \includegraphics[scale=1]{pics/GT_headshot.jpg} 
       \end{center}
       }
       \subcolumn{3} \block[bodyoffsetx=-4cm,bodyverticalshift=-5cm]{}{
       \begin{center}
       \includegraphics[scale=1.2]{pics/unimelb-logo.jpg}
      \end{center}
       }
       \subcolumn{3} \block[bodyoffsetx=-5cm,bodyverticalshift=-6cm]{}{
       \begin{center}
       \includegraphics[scale=1]{pics/tskit-logo.png}
      \end{center}
       }
 \end{subcolumns}
 \end{columns}         

 \end{document}


\endinput
%%
%% End of file `tikzposter-example.tex'.
